config/gradle/jre.gradle

```diff
@@ -0,0 +1,46 @@
+enum Jre {
+    // TODO Use `.tar.gz` on Linux
+    Linux32("zip"),
+    Linux64("zip"),
+    // TODO rename `Mac` to `MacOS` when artifact is renamed: https://github.com/MovingBlocks/TerasologyJRE/pull/5
+    Mac("zip"),
+    Windows32("zip"),
+    Windows64("zip");
+
+    public URL downloadUrl;
+    public File targetArchive;
+
+    Jre(String extension) {
+        String baseUrl = "http://artifactory.terasology.org/artifactory/libs-release-local"
+
+        // TODO can we use gradle dependencies/configurations to download this for the bundling task?
```

Comment:
Definitely, and then we'd also get `UP-TO-DATE` when Gradle runs again and the JREs are already present. I think. My Gradle-fu is still mostly at the [Just chuck bits in until it works](https://quickbirdstudios.com/blog/gradle-kotlin-buildsrc-plugin-android/) level

Reply:
Looks like I should read this article eventually. 😅 The behavior of the download plug-in is not too bad, it won't download again unless the file has changed on the server (not that this should ever be the case without changing version, but ...). 
@jdrueckert and me looked into configurations but could not figure it out properly... =/